### ISSUE TITLE
Search (hardware) button starts web search, not foursquare search.

### ISSUE THREAD

**Issue by GoogleCodeExporter** [iamakshayv/foursquared#60]
```
It doesn't work.

What steps will reproduce the problem?
1.
2.
3.

What is the expected output? What do you see instead?


What version of the product are you using? On what operating system?


Please provide any additional information below.

```

Original issue reported on code.google.com by `jlapenna@google.com` on 28 Aug 2009 at 6:58

**Comment by GoogleCodeExporter** [comment]
```
Issue 63 has been merged into this issue.
```

Original comment by `jlapenna` on 29 Aug 2009 at 6:18

**Comment by GoogleCodeExporter** [comment]
Original comment by `jlapenna` on 29 Aug 2009 at 6:19
- Changed title: **Search (hardware) button starts web search, not foursquare search.**

**Comment by GoogleCodeExporter** [comment]
Original comment by `jlapenna` on 29 Aug 2009 at 6:22
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision 2d126b816a.
```

Original comment by `jlapenna` on 29 Aug 2009 at 7:21
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision 2d126b816a.
```

Original comment by `jlapenna` on 5 Jan 2010 at 5:47
